src/Numerical/Spectral/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 # See LICENSE.txt for details.
 
 set(MY_LIBRARY_SOURCES
+    Spectral/DefiniteIntegral.cpp
     Spectral/LegendreGaussLobatto.cpp)
```

Comment:
It'll cause fewer conflicts going forward if the `)` is moved to it's own line (my fault, not yours)